dataprep/connector/__init__.py

```diff
@@ -4,8 +4,17 @@
 
 from .connector import Connector
 from .generator import ConfigGenerator, ConfigGeneratorUI
+from ..utils import display_dataframe
+from .info import info
 
-__all__ = ["Connector", "ConfigGenerator", "ConfigGeneratorUI", "connect"]
+__all__ = [
+    "Connector",
+    "ConfigGenerator",
+    "ConfigGeneratorUI",
+    "connect",
+    "display_dataframe",
```

Comment:
Is this display_dataframe used anywhere in this PR?

Reply:
I thought it was but it's not, good catch. I'll remove the code for it and make a separate PR for display_dataframe once this PR has been accepted.

Reply:
Great!